package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@nmshd/connector",
-    "version": "3.11.2",
```

Comment:
1. branch says release/v5, please change the branch name
2. as long as the runtime runs under prereleases we have to also publish prereleases here (`4.0.0-alpha.1` see https://github.com/nmshd/runtime/pull/146)

Reply:
(same for the SDK)